lib/devices/android/bootstrap/src/io/appium/android/bootstrap/utils/ReflectionUtils.java

```diff
@@ -19,7 +19,6 @@ public static boolean clearAccessibilityCache() {
     boolean success = false;
 
     try {
-      final ReflectionUtils utils = new ReflectionUtils();
```

Comment:
this is some cruft--no need for this any longer since everything in this class is static now.